test/visibility/import/edgeCases/multipleDef.good

```diff
@@ -0,0 +1 @@
+4
```

Comment:
I think this should be a conflict, I think the re-exported symbol should be considered at the same depth as the one declared in B.  But it'd maybe be good to have more people weigh in

Reply:
That might imply that this should be part of the calculation for "scope->getField"?

Reply:
Okay, I take back that last comment (but I can't see a way to delete it, weirdly enough)